src/config.rs

```diff
@@ -37,6 +37,35 @@ impl Default for SortOrder {
     }
 }
 
+#[derive(Debug, Eq, PartialEq, Hash, Copy, Clone)]
+#[derive(Serialize, Deserialize)]
+#[serde(deny_unknown_fields)]
+pub enum SassOutputStyle {
+    Nested,
+    Expanded,
+    Compact,
+    Compressed,
+}
+
+#[derive(Debug, PartialEq)]
+#[derive(Serialize, Deserialize)]
+#[serde(deny_unknown_fields)]
+pub struct SassOptions {
+    pub sass_dir: String,
+    pub extensions: Vec<String>,
```

Comment:
How come this needs to be configurable?

Reply:
Well, I come from the world of multimedia where rule number one is: if you know the file name extension, you know _nothing_ (what is in an `avi` or `mov` again? Can you play an `mp4`?). There, unless you read its first bytes until you detect something, you have no idea what you are dealing with. With that background I always feel super insecure with coding something that purely relies on file path extensions for feature or file type detection (as it is bullocks), and therefore I have the urge to _at least_ allow the user to reconfigure it, if they also chose to rename the extensions (even if that is not common place).

But especially with the argument of `scss` and `sass` I make later, there is a fair point in arguing that no one really needs or wants to configure this ever ... I most certainly never did and don't see a good reason to do so in the future. I am fine with removing it and setting it to `scss` only.

Reply:
Nice thing is it can always be added later if needed.

Reply:
Agreed. Will simplify it.